### DISCUSSION TITLE
Add public Grafana snapshot for reference

### DISCUSSION THREAD

**Discussion by Otto-AA** [paradigmxyz/reth#8283] [Ideas]
I think it would be nice to link to a public Grafana snapshot in the [Observability chapter](https://paradigmxyz.github.io/reth/run/observability.html).

This could be helpful for two scenarios:
1. to be able to compare against a "normal" syncing process (and find potential anomalies, instead of guessing if this number is always so high/low in stage ABC)
2. to have an idea of the benefits before setting Prometheus+Grafana up

I'm not sure how to do this best, as the snapshot feature only saves the *visible* panels. The furthest I got was to zoom out and change to a horizontal screen to get ~60% of the panels :upside_down_face:

**Comment by emhane** [comment]
agreed, though we simply link to the public dashboards for live updates https://reth.paradigm.xyz/d/2k8BXz24x/reth?orgId=1&refresh=30s, feel free to open a pr
